examples/lusid/run-valuation/2-create-portfolio.sql

```diff
@@ -0,0 +1,14 @@
+-- Defining scope and code variables
+@@portfolioScope = select 'ibor';
+@@portfolioCode1 = select 'EQUITY_UK';

```

Comment:
Should this be 'IBOR' and 'uk-equity' to match the scope and code in create-transaction-portfolio.sql? Or are they intentionally different?